numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2301,6 +2336,13 @@ def add_newdoc(place, name, doc):
     >>> np.logical_or(x < 1, x > 3)
     array([ True, False, False, False,  True])
 
+    If the inputs are ndarrays, then np.logical_or is equivalent to '|'.
+
+    >>> a = np.array([True,False])
+    >>> b = np.array([False,False])
+    >>> a|b
```

Comment:
As above, this is `bitwise_or`

Reply:
I'll make the changes to these both functions.